src/ikrpg.js

```diff
@@ -365,7 +360,7 @@ Hooks.on("updateCombat", (combat, changed) => {
 });
 
 
-Hooks.on("renderChatMessage", (message, html, data) => {
+Hooks.on("renderChatMessageHTML", (message, html, data) => {
```

Comment:
This change actually makes spells stop working on my version of foundry. version 12 build 331